language/Portuguese_Brazil/strings.po

```diff
@@ -426,19 +431,19 @@ msgstr "Opções de Performance"
 
 #: src/frontend/mame/ui/submenu.cpp:52
 msgid "Auto frame skip"
-msgstr "Salto de quadro Automático"
+msgstr "Auto frameskip"
 
 #: src/frontend/mame/ui/submenu.cpp:53
 msgid "Frame skip"
-msgstr "Salto de quadro"
+msgstr "Frame skip"
 
 #: src/frontend/mame/ui/submenu.cpp:54
 msgid "Throttle"
 msgstr "Suprimir"
 
 #: src/frontend/mame/ui/submenu.cpp:55
 msgid "Mute when unthrottled"
-msgstr ""
+msgstr "Mudo quando desacelerado"
```

Comment:
I’d like to confirm this is translated correctly.  This option causes audio to be muted when the system is allowed to run as fast as possible (i.e. when it is not being limited to normal speed).  Is this how you understood it when translating?

Reply:
nope it's not, there should be a comment about this since this has a double meaning, thanks for warning me. :)